### ISSUE TITLE
Issue of RUNNING ANALYSIS OF CHROMOSOMES ...

### ISSUE THREAD

**Issue by mimingcui** [esrud/GONE#29]
[Archive.zip](https://github.com/esrud/GONE/files/13230022/Archive.zip)
Hi,
When I run the linux script, it stucks at `RUNNING ANALYSIS OF CHROMOSOMES ...` forever.
It would be greatly appreciated if you can look into the attached data and parameter. 
Thanks!
Ming

**Comment by armando-caballero** [comment]
In your ped file, each line should be an individual:
1 ALB2019_concat_A10.sort 0 0 0 -9 A A T T C C T T G G C C G G A A A A A A A A C C A A C C T T G G T T G G T T G G C C A A T T G G C C G G G G C C C C A A A A A A A A G G G G G G A A G G C C G G T T T C T C C G C A C T A A G G C C C C
1 ALB2019_concat_A11.sort 0 0 0 -9 A G T T C T T C G G C C G G A A A A A A A A C C A G C C T T G A T T G A T T A G C C A A T T G G C CG G G T C C C T G A A A A G G G G G G G A G A A G G C T G G T T C C C C G G A A T T A A C C C C T T
1 ALB2019_concat_A12.sort 0 0 0 -9   --------------------------------
Try that.
Armando

**Comment by armando-caballero** [comment]
I run GONE with you files (after correcting the format in the ped file) and the software run ok.
The output is below (with a lot of noise because you have only 50 SNPs in the example). The ped file I used is also below.

Ne averages over 40 independent estimates.
Generation	Geometric_mean
1	66.412
2	66.412
3	66.412
4	66.412
5	50661
6	50661
7	50661
8	50661
9	52281.8
10	53976.7
11	53074.2
12	53074.2
13	49936.8
14	43668.5
15	44270.6
16	44270.6
17	41331.7
18	6181.05
19	4737.6
20	4737.6
21	3608.34
22	141.337
23	117.046
24	117.046
25	120.529
etc.

data.ped

0 ALB2019_concat_A10.sort 0 0 0 -9 A A T T C C T T G G C C G G A A A A A A A A C C A A C C T T G G T T G G T T G G C C A A T T G G C C G G G G C C C C A A A A A A A A G G G G G G A A G G C C G G T T T C T C C G C A C T A A G G C C C C
0 ALB2019_concat_A11.sort 0 0 0 -9 A G T T C T T C G G C C G G A A A A A A A A C C A G C C T T G A T T G A T T A G C C A A T T G G C C G G G T C C C T G A A A A G G G G G G G A G A A G G C T G G T T C C C C G G A A T T A A C C C C T T
0 ALB2019_concat_A12.sort 0 0 0 -9 G G T T T T C C G G C C G G A A A A A A A A C C A A C C T T G G T T G G T T G G C C A A T T G G C C G G G G C C C C A A A A G G G G G G G G G G A A G G C C G G T T C C C C G G A A T T A A C C C C T T
0 ALB2019_concat_A13.sort 0 0 0 -9 G G T T T T C C G G C C G G A A A A A A A A C C G G C C T T A A T T A A T T A A C C A A T T G G C C G G T T C C T T G G A A A G G G G G G G A G A A G G C T G G T T C C C C G G A A T T A A C C C C T T
0 ALB2019_concat_A14.sort 0 0 0 -9 A G T T C T T C G G C C A G A A A A A A A A A C A G C C T T A A T T A A C T A G C C A A T T G G C C G G G T C C T T G A A A A G A G G G G G G G A A G G C T A G T T T C T C C G C A C T A A G G C C C C
0 ALB2019_concat_A15.sort 0 0 0 -9 A G T T C T C C G G C C A G A A A A A A A A A C A G C C T T A A T T A A C T A G C C A A T T G G C C G G G T C C T T G A A A A G A G G G G G G G A A G G C T A G T T T C T C C G C A C T A A G G C C C C
0 ALB2019_concat_A16.sort 0 0 0 -9 G G T T T T C C G G C C G G A A A A A A A A C C G G C C T T A A T T A A T T A A C C A A T T G G C C G G T T C C T T G G A A G G G G G G G G G G A A G G C T A G T T C C C C C G C A C T A A G C C C T C
0 ALB2019_concat_A17.sort 0 0 0 -9 G G T T T T C C G G C C G G A A A A A A A A C C G G C C T T A A T T A A T T A A C C A A T T G G C C G G T T C C T T G G A A A A G G G G G G A A A A G G T T G G T T C C C C G G A A T T A A C C C C T T
0 ALB2019_concat_A18.sort 0 0 0 -9 G G T T T T C C G G C C G G A A A A A A A A C C G G C C T T A A T T A A T T A A C C A A T T G G C C G G T T C C T T G G A A G G G G G G G G G G A A G G T T A A T T C C C C C C C C C C A A G G C C C C 
0 ALB2019_concat_A19.sort 0 0 0 -9 G G T T T T C C G G C C G G A A A A A A A A C C G G C C T T A A T T A A T T A A C C A A T T G G C C G G T T C C T T G G A A A G G G G G G G A G A A G G T T A G T T C C C C C G C A C T A A G C C C T C 
0 ALB2019_concat_A20.sort 0 0 0 -9 G G T T T T C C G G C C G G A A A A A A A A C C G G C C T T A A T T A A T T A A C C A A T T G G C C G G T T C C T T G G A A G G G G G G G G G G A A G G T T A A T T C C C C C C C C C C A A G G C C C C 
0 ALB2019_concat_A21.sort 0 0 0 -9 A G T T C T T C G G C C G G A A A A A A A A C C A G C C T T G A T T G A T T A G C C A A T T G G C C G G G T C C C T G A A A G G G G G G G G G G A A G G C T A G T T C C C C C G C A C T A A G C C C T C 
ETC ETC ETC

**Comment by mimingcui** [comment]
Hi, did you also use the attached parameter file? 

I still couldn't run it and had the same issue:
```
bash script_GONE.sh data
cp: 'data.map' and 'data.map' are the same file
cp: 'data.ped' and 'data.ped' are the same file
DIVIDE .ped AND .map FILES IN CHROMOSOMES
RUNNING ANALYSIS OF CHROMOSOMES ...
```
Thanks.

**Comment by armando-caballero** [comment]
No, I simply corrected your initial ped file as indicated above. I removed it after using it. You have an error of format in your file
Armando.
